V2/gameFuseFriendRequest.js

```diff
@@ -0,0 +1,169 @@
+class GameFuseFriendRequest {
+    constructor(friendshipId, requestedAt, otherUser) {
+        this.friendshipId = friendshipId
+        this.requestedAt = requestedAt;
+        this.otherUser = otherUser;
+    }
+
+    getOtherUser() {
+        return this.otherUser;
+    }
+
+    getFriendshipID() {
+        return this.friendshipId;
+    }
+
+    getRequestedAt() {
+        return this.requestedAt;
+    }
+
+    static async send(username, callback = undefined) {
```

Comment:
should this be call `send` or sendFriendRequest? I called it send because I thought it might be redundant "GameFuseFriendRequest.sendFriendRequest", but maybe for consistency it should be the same as the sendFriendRequest instance method inside GameFuseUser?

Reply:
I think it can be send.  some_one_i_admire.sendFriendRequest makes sense because we are applying it to a user, but send applied to a GameFuseFreindRequest makes more sense. I like .send